test/extended/oauth/requestheaders.go

```diff
@@ -504,32 +508,3 @@ func getTokenFromResponse(resp *http.Response) string {
 
 	return ""
 }
-
-func waitForAuthenticationProgressing(oc *exutil.CLI, expectedProgressing configv1.ConditionStatus) {
-	err := wait.PollImmediate(time.Second, 10*time.Minute, func() (bool, error) {
-		authn, err := oc.AdminConfigClient().ConfigV1().ClusterOperators().Get(context.Background(), "authentication", metav1.GetOptions{})
-		if err != nil {
-			e2e.Logf("Error getting authentication operator: %v", err)
-			return false, err
-		}
-
-		progressing := clusteroperatorhelpers.FindStatusCondition(authn.Status.Conditions, configv1.OperatorProgressing)
-		if progressing == nil || progressing.Status != expectedProgressing {
-			e2e.Logf("Waiting for progressing condition to be %q: %s", expectedProgressing, spew.Sdump(authn.Status.Conditions))
-			return false, nil
-		}
-
-		if expectedProgressing == configv1.ConditionFalse {
```

Comment:
[30 minutes](https://github.com/openshift/library-go/blob/master/pkg/operator/apiserver/controllerset/apiservercontrollerset.go#L151-L168=). But anyway, After reading the docs, I tend to agree with you.

It will be interesting to see if there is any change in flakiness afterwards.